images/base/Dockerfile

```diff
@@ -24,12 +24,14 @@ ARG BASE_IMAGE=debian:bookworm-slim
 FROM $BASE_IMAGE as base
 
 # copy in static files
-# all scripts are 0755 (rwx r-x r-x)
+# all scripts and directories are 0755 (rwx r-x r-x)
 # all non-scripts are 0644 (rw- r-- r--)
 COPY --chmod=0755 files/usr/local/bin/* /usr/local/bin/
 
-COPY --chmod=0644 files/kind/ /kind/
-COPY --chmod=0755 files/kind/bin/* /kind/bin/
+# Don't specify a chmod for /kind itself, else /kind/bin ends up with 0644
```

Comment:
this isn't suitable, because then the build depends on umask.

we'll need another workaround, e.g. `RUN chmod` unfortunately

Reply:
Sorry about the delay, using a chmod now.